test/configCases/target/webworker/index.js

```diff
@@ -0,0 +1,110 @@
+require("should");
```

Comment:
Couldn't we just `require()` that file from `../web/index.js`. It would be good to minimize the amount of files that need to be refactored if anything changes.